azure-pipelines-template.yml

```
@@ -39,7 +39,7 @@ jobs:
     displayName: Install dependencies
   - script: npm test
     displayName: Run tests
-  - script: npm run prebuild
+  - script: npm run prebuild --arch=$(parameters.arch)
```

I see "line 1: parameters.arch: command not found" in the build...

```suggestion
  - script: npm run prebuild --arch=${{ parameters.arch }}
```